app/controllers/api/v0/manifests_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V0::ManifestsController < ApplicationController
   def search
     if !params[:q] && !params[:aq]
-      render json: {}, status: 400
+      manifests = Manifest.all
```

Comment:
I suspect you changed this when you were trying to use the API via external HTTP request from the manifests controller?